Algebra.java

```diff
@@ -26,42 +26,178 @@ public static void main(String args[]) {
 	// Returns x1 + x2
 	public static int plus(int x1, int x2) {
 		// Replace the following statement with your code
-		return 0;
+		int sum = x1;
+		if (x2>=0)
+		{
+			for (int i=0; i<x2;i++)
+			{
+				sum++;
+			}
+		}
+		else
+		{
+			for ( int j=0; j>x2;j--)
```

Comment:
Spaces...
Please go through the Java style guidelines document in the course's Moodle page.